tests/categorical_test.py

```diff
@@ -267,23 +267,24 @@ def testSaveAndLoadCategorical(self):
         """
         num_elems = 51  # _getCategorical starts counting at 1, so the size is really off by one
         cat = self._getCategorical(size=num_elems)
-        with self.assertRaises(ValueError):  # Expect error for mode not being append or truncate
-            cat.save("foo", dataset="bar", mode="not_allowed")
+        with self.assertRaises(ValueError):
+            # Expect error for mode not being append or truncate
+            cat.to_hdf("foo", dataset="bar", mode="not_allowed")
 
         with tempfile.TemporaryDirectory(dir=CategoricalTest.cat_test_base_tmp) as tmp_dirname:
             dset_name = "categorical_array"  # name of categorical array
 
             # Test the save functionality & confirm via h5py
-            cat.save(f"{tmp_dirname}/cat-save-test", dataset=dset_name)
+            cat.to_hdf(f"{tmp_dirname}/cat-save-test", dataset=dset_name)
```

Comment:
Should we also add a test for `to_parquet`?

Reply:
@joshmarshall1 I have this was equivalent here. Parquet testing has its own file, so it would be duplicate here